.github/workflows/fbgemm_docs.yml

```diff
@@ -63,6 +63,10 @@ jobs:
       run: |
         conda run -n build_binary \
           python -c "import fbgemm_gpu"
+    - name: Install Doxygen
+      run: |
+        sudo apt-get install doxygen
```

Comment:
Do we need doxygen on the system level or "conda install -n build_binary -c conda-forge doxygen" would be enough?

Reply:
On the env level should be enough!